src/vs/workbench/electron-browser/commands.ts

```diff
@@ -423,3 +424,16 @@ export function registerCommands(): void {
 		});
 	});
 }
+
+namespace GlobalQuickOpenCommand {
+	export const ID = 'vscode.quickOpen';
```

Comment:
Yes, that's what I meant. Nuke the action, the ID is now slight misleading but we don't wanna break existing keybinding customisations.